tangelo/tangelo/server.py

```diff
@@ -541,6 +541,10 @@ def dirlisting(dirpath, reqpath):
         return result
 
     def execute_analysis(self, query_args):
+        # Hide the identity/version number of the server technology in the
+        # response headers.
+        cherrypy.response.headers["Server"] = "Tangelo"
```

Comment:
The reason I'm making this change in the first place is that revealing the version number was deemed to be a (minor) security problem, so probably not :-/.

However, it may be possible to do this in a serverside API function. Then, using that function to create a service that shows it would still be a security problem, but one under complete control of the application developer.